### ISSUE TITLE
Step 3 : No CardBoard under Component

### ISSUE THREAD

**Issue by jeancdc** [googlesamples/io2015-codelabs#23]
In the step 3, "From the menu, click  Component > Cardboard > Update Stereo Cameras."

There is no CardBoard in the Component menu...

**Comment by jainankit** [comment]
Yes I noticed the same, how did you fix this?

**Comment by claywilkinson** [comment]
Try Assets > Refresh and see if it shows up, otherwise make sure the console does not have any error messages (you'll need to fix them if there are errors), finally try to re-import the package.

**Comment by jainankit** [comment]
Yes thanks, I actually saved and then restarted Unity and the option showed up (perhaps worth mentioning in the tutorial).
I also am having trouble going into VR mode. I completed all things from Step5 but still the view appears same. I did run the application with both Cardboard on and without. The Unity preview also doesn't show the two camera views. What am I missing?

**Comment by jainankit** [comment]
Oh I figured it out . I do have to click the Cardboard mode button first, to get a menu with Cardboard Mode. I don't understand very well though coz the CardboardModeMgr script was supposed to start in Cardboard mode (atleast that's what I understood from Code and documentation).

**Comment by claywilkinson** [comment]
Great! glad you figured it out.  I'll make a note to update the documentation to make it more clear.

**Comment by BalbretOK** [comment]
It was hard for me to find the Update Stereo Cameras too. No matter what I did, I was unable to find it under the menu bar (Component > Cardboard was never there).

But in the end I found it at property inspector for the Camera component. Click the gear (settings) and the 'Update Stereo Cameras' is there!
